packages/amplify_core/lib/src/types/storage/storage_path_with_identity_id.dart

```diff
@@ -0,0 +1,20 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
*todo:* remove. This looks like the result of a merge conflict after this file was renamed.